tests/Jira/ApiTest.php

```diff
@@ -25,4 +25,102 @@ public function testSetEndpointTrailingSlash()
         $api->setEndPoint($url);
         $this->assertEquals($url, $api->getEndpoint());
     }
+
+    /**
+     * Tests that the updateVersion call constructs the correct api call
+     */
+    public function testUpdateVersion()
+    {
+        $params = array(
+            'released' => true,
+            'releaseDate' => '2010-07-06',
+        );
+
+        // Stub the api method and keep the rest intact
+        /** @var Api|\PHPUnit_Framework_MockObject_MockObject $api */
+        $api = $this->getMockBuilder('\chobie\Jira\Api')->setMethods(array('api'))->disableOriginalConstructor()->getMock();
```

Comment:
This test doesn't really test anything except out internal cross-class interactions 😄 

You can keep it of course, but ideally I would like to see #74 implemented at some point.